contributing/developing.md

```diff
@@ -180,11 +180,23 @@ rules](https://palantir.github.io/tslint/usage/type-checking/) which requires a
 tsconfig.json to be passed to tslint.
 
 ### `npm run check:links`
+
 Runs `markdown-link-check` on all markdown files in the repo to check for any broken links.
+
 - Does not check html files.
 - Ignores [429 Too Many Requests](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429)
   - We get 429 mostly from github as there are many URLs pointing to PRs etc in Changelog
 
 ### `npm run check:deps`
 
 This runs `depcheck` on each package to check for unused and missing dependencies. Pay particular attention to "Missing dependencies". Unused dependency result might have [false positives](https://github.com/depcheck/depcheck#false-alert). Check code usage to verify.
+
+## Node Configuration
+
+### .npmrc
+
+The npmrc allows for project-level [configuration](https://docs.npmjs.com/cli/v8/using-npm/config) of the npm environment. The settings that we care about are `engine-strict` (this enforces the minimum node version in our packages) and `save-exact` (saving exact versions of dependencies from the package.json).
+
+### .nvmrc
+
+Our nvmrc specifies the minimimum node version required to run the project.
```

Comment:
cool thanks

Reply:
Can the rc files have inline comments - if yes would it make sense to have these in the files themselves for better context and future proof / sync ?